app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountForm/Step1_CreateAccountFormContainer.jsx

```diff
@@ -164,7 +156,7 @@ class CreateAccountFormContainer extends Component {
 			if (success) {
 				// User is automatically logged in, and redirected to the logged in view of BrowserCreateAccountForm
 				actions.getUser().then(() => {
-					if (isUpdatesChecked) actions.handleEmailPreferencesCheckboxChange('global', isUpdatesChecked);
+					actions.subscribeToEmailList('global');
```

Comment:
See above - we may not need this at all?

Reply:
See comment above